src/TC/src/FfmpegDecoder.cpp

```diff
@@ -295,8 +275,6 @@ struct FfmpegDecodeFrame_Impl {
       return SaveYUV422(frame);
     case AV_PIX_FMT_YUV444P:
       return SaveYUV444(frame);
-    case AV_PIX_FMT_GRAY12LE:
-      return SaveGRAY12LE(frame);
```

Comment:
Same here, I think this should not be removed.

Reply:
Yes, you're right.
The reason is same as above.